lib/util.js

```diff
@@ -374,5 +375,6 @@ module.exports = {
   toUSVString,
   TextDecoder,
   TextEncoder,
-  types
+  types,
+  ERR_INVALID_STATE
```

Comment:
This isn't necessary for the purposes of benchmarking. Benchmarks can specify flags like `--expose-internals`, which should be enough to get at the internal error objects.